### PR TITLE
Add project_create to PagureService and add tests for it

### LINKED ISSUE #204: Implement Pagure method for create_project
```python
    def project_create(self, repo: str, namespace: str = None) -> "PagureProject":
        pass
```

API documentation:
https://pagure.io/api/0/#projects

AC:

- [ ] implement the method in `PagureService`
- [ ] create at least two tests for that (with and without specifying `namespace`)

### PR DESCRIPTION
**Pull request by mfocko**
Closes #204

### CODE REVIEW

File: ogr/services/pagure/service.py [hunk 1]
```diff
@@ -244,5 +245,15 @@ def project_create(
...
-        self.call_api(request_url, "POST", data=parameters)
+        try:
+            self.call_api(request_url, "POST", data=parameters)
+        except PagureAPIException as ex:
+            if (
+                ex.pagure_response
+                and ex.pagure_response["errors"]["namespace"][0] == "Not a valid choice"
+            ):
+                raise OgrException(
+                    f"Cannot create project in given namespace ({namespace})."
```
Comment (lachmanfrantisek) [inline ogr/services/pagure/service.py]: Can we determine these two possibilities?

- non-existing namespace
- no permissions

Comment (mfocko) [reply ogr/services/pagure/service.py]: I remember it returning the same thing. _not a valid choice_

Comment (mfocko) [reply ogr/services/pagure/service.py]: yeah, returns the same thing

Comment (lachmanfrantisek) [reply ogr/services/pagure/service.py]: Hmm. I'll create an issue for that, so we don't forget about that.

Comment (lachmanfrantisek) [reply ogr/services/pagure/service.py]: #245

**Pull request merged**